Please support go 1.16

`go` 1.16 or later is required to build binaries for Macs with Apple's M1 chip but Variant does not build with go 1.16.

```
$ go version
go version go1.16.3 linux/amd64
$ variant export binary cli cli
go: github.com/mumoshu/variant2@v0.37.1: missing go.sum entry; to add it:
	go mod download github.com/mumoshu/variant2
Error: command "sh -c cd /tmp/variant-atmos886148816; go build -o /localhost/CP_dev/atmos/abin /tmp/variant-atmos886148816": exit status 1
```
 If I try to fix that, it still fails, although differently
```
$ variant export go cli build
$ cd build
$ go mod download
$ go build
# github.com/mumoshu/variant2/pkg/app
/conf/go/pkg/mod/github.com/mumoshu/variant2@v0.37.1/pkg/app/app.go:611:36: cannot use TestDeps{} (type TestDeps) as type testing.testDeps in argument to testing.MainStart:
	TestDeps does not implement testing.testDeps (missing SetPanicOnExit0 method)
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -6,7 +6,7 @@ jobs:
     strategy:
       matrix:
         go:
-        - 1.15.x
+        - 1.16.x
     name: Go ${{ matrix.go }} test
     steps:
     - uses: actions/checkout@master
@@ -32,7 +32,7 @@ jobs:
     strategy:
       matrix:
         go:
-        - 1.15.x
+        - 1.16.x
     name: Go ${{ matrix.go }} smoke test
     steps:
     - uses: actions/checkout@master
@@ -55,7 +55,7 @@ jobs:
     strategy:
       matrix:
         go:
-        - 1.15.x
+        - 1.16.x
     name: Go ${{ matrix.go }} lint
     steps:
     - uses: actions/checkout@master
@@ -77,7 +77,7 @@ jobs:
       name: Set up Go
       uses: actions/setup-go@v1
       with:
-        go-version: 1.15.x
+        go-version: 1.16.x
     -
       name: Set goreleaser .Env
       run: |

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -16,7 +16,7 @@ jobs:
       name: Set up Go
       uses: actions/setup-go@v1
       with:
-        go-version: 1.15.x
+        go-version: 1.16.x
     -
       name: Set goreleaser .Env
       run: |

--- a/pkg/app/testdeps_deps.go
+++ b/pkg/app/testdeps_deps.go
@@ -55,6 +55,8 @@ func (TestDeps) WriteProfileTo(name string, w io.Writer, debug int) error {
 	return pprof.Lookup(name).WriteTo(w, debug)
 }
 
+func (TestDeps) SetPanicOnExit0(bool) {}
+
 // ImportPath is the import path of the testing binary, set by the generated main function.
 var ImportPath string
 

